docs/resources.rst

```diff
@@ -5,58 +5,63 @@ Resources
-* `Awesome Solidity <https://github.com/bkrem/awesome-solidity>`_
+* `Solidity website <https://soliditylang.org/>`_
+* `Solidity changelog <https://github.com/ethereum/solidity/blob/develop/Changelog.md>`_
+* `Solidity codebase on GitHub <https://github.com/ethereum/solidity/>`_
+* `Solidity language users chat <https://matrix.to/#/#ethereum_solidity:gitter.im>`_
+* `Solidity compiler developers chat <https://matrix.to/#/#ethereum_solidity-dev:gitter.im>`_
+* `awesome-solidity <https://github.com/bkrem/awesome-solidity>`_
 * `Solidity by Example <https://solidity-by-example.org/>`_
-* `Solidity Documentation Community Translations <https://github.com/solidity-docs>`_
+* `Solidity documentation community translations <https://github.com/solidity-docs>`_
 
 Integrated (Ethereum) Development Environments
 ==============================================
 
-    * `Brownie <https://eth-brownie.readthedocs.io/en/stable/>`_
-        Python-based development and testing framework for smart contracts targeting the Ethereum Virtual Machine.
+* `Ape <https://docs.apeworx.io/ape/stable/az>`_
+        A Python-based web3 development tool for compiling, testing, and interacting with smart contracts.
 
-    * `Dapp <https://dapp.tools/>`_
+* `Brownie <https://eth-brownie.readthedocs.io/en/stable/>`_
+        A Python-based development and testing framework for smart contracts targeting the Ethereum Virtual Machine.
+        💡 Note: As per the official docs, Brownie is no longer actively maintained.
+        Future releases may come sporadically - or never at all.
+        Check out Ape Framework (first in list) for all your python Ethereum development needs.
+
+* `Dapp <https://dapp.tools/>`_
         Tool for building, testing and deploying smart contracts from the command-line.
 
-    * `Foundry <https://github.com/foundry-rs/foundry>`_
+* `Foundry <https://github.com/foundry-rs/foundry>`_
         Fast, portable and modular toolkit for Ethereum application development written in Rust.
 
-    * `Hardhat <https://hardhat.org/>`_
+* `Hardhat <https://hardhat.org/>`_
         Ethereum development environment with local Ethereum network, debugging features and plugin ecosystem.
 
-    * `Remix <https://remix.ethereum.org/>`_
+* `Remix <https://remix.ethereum.org/>`_
         Browser-based IDE with integrated compiler and Solidity runtime environment without server-side components.
 
-    * `Truffle <https://trufflesuite.com/truffle/>`_
+* `Truffle <https://trufflesuite.com/truffle/>`_
         Ethereum development framework.
+        💡 Note: Consensys announced the sunset of Truffle on September 21, 2023.
+        Current users may check out the migration path and available product support `here.
+        <https://consensys.io/blog/consensys-announces-the-sunset-of-truffle-and-ganache-and-new-hardhat>`_
 
 Editor Integrations
 ===================
 
 * Emacs
-
     * `Emacs Solidity <https://github.com/ethereum/emacs-solidity/>`_
```

Comment:
Please make sure this renders correctly before merging. In RST there must be an empty line between a bullet list and the surrounding text. If it's not there, the list gets mangled into a single paragraph as if it was a continuation of the previous sentence. Not sure if it is needed in this particular case, but if it was here already, it might have been for this very reason.